industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jak1/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,14 @@
+  # Preparation done in Schrodinger suite 2024-2 (patch 1)
+  # Read in jak1
+  # Contains two PTR residues, converted to Tyr with standard Maestro mutate
+  # Ran diagnostics, has two orientations for His885.  Select default (53% occ) and commit.
+  # this His has the proton on E but is charged on D with no proton.
```

Comment:
This is likely fine, so I don't recommend making any changes. However I'm so sorry I don't think I fully understand what is meant here, we will likely ask for further clarification when it comes to writing the manuscript.